doc/source/contributing.rst

```diff
@@ -137,15 +137,63 @@ following the steps in the `Creating a New Pull Request
 section <#creating-a-new-pull-request>`__.
 
 
+Opening Issues
+~~~~~~~~~~~~~~
+Should you come across a bug in ``PyMAPDL`` or otherwise encounter some
+unexpected behaviour you should create an "issue" regarding it. Issues are
+created and submitted `here <https://github.com/pyansys/pymapdl/issues>`_.
+Issues are used when developing to keep track of what is being
+worked on at any one time, and by who. We have two issue templates
+that we recommend you use:
+
+* Bug report template
+* Feature request template
+
+If your issue does not fit into these two categories you are free
+to create your own issue as well.
+
+Issues should contain sufficient context for others to reproduce your
+problem, such as the application versions you are using as well as
+reproduction steps. Use issue labels like "Documentation" to further
+highlight your issue's category.
+
+Developers will respond to your issue and hopefully resolve it! Users
+are encouraged to close their own issues once they are completed.
+Otherwise, issues will be closed after a period of inactivity at the
+discretion of the maintainers.
+
+Should it turn out the fix did not work, or your issue was closed
+erroneously, please re-open your issue with a comment addressing why.
+
+Open ended questions should go in the `discussions tab <https://github.com/pyansys/pymapdl/discussions>`_
+and should an issue generate additional discussion, further issues
+should be spun out into their own separate issues. This helps developers
+keep track of what is being done and what needs to be done.
+
+
+Discussions
+~~~~~~~~~~~
+
+General questions about PyMAPDL should go in the
+`discussions tab <https://github.com/pyansys/pymapdl/discussions>`_ on
+the repo rather than as issues themselves. Issues can be spun out of
```

Comment:
```suggestion
General questions about PyMAPDL should be raised in
`Discussions <https://github.com/pyansys/pymapdl/discussions>`_ in
this repository rather than as issues themselves. Issues can be spun out of
```